### PR TITLE
fix

### PR DESCRIPTION
**Pull request by YosefGul**

### CODE REVIEW

**Review by spektra2147 — APPROVED** [review]

File: addons/default/visiosoft/advs-module/resources/views/list/gallery.twig [hunk 1]
```diff
@@ -5,48 +5,37 @@
...
-            <div class="w-50 gallery-item-wrapper mb-1{{ adv.doping != null ? ' doping_type4' }}">
-                <div class="gallery-item rounded p-3 d-flex h-100">
-                    <div>
-                        <a href="{{ adv.detail_url }}">
-                            <img src="{{ adv.cover_photo }}" alt="{{ adv.name }}" class="rounded">
-                        </a>
-                        <div class="action-data-wrapper d-flex justify-content-center">
-                            <div class="action-data d-flex flex-column">
-
-                                {{ addBlock('list/extra-actions', {'ad': adv, 'vars': _context}, ['favs', 'comparisons'])|raw }}
```
Comment (muammertop) [inline addons/default/visiosoft/advs-module/resources/views/list/gallery.twig:22]: revert to addblock

File: addons/default/visiosoft/advs-module/resources/views/list/gallery.twig [hunk 2]
```diff
@@ -5,48 +5,37 @@
...
+                            <img class="product-image-in" src="{{ adv.cover_photo }}" alt="{{ adv.name }}">
+                        </div>
+                        <div class="product-info-hb d-flex flex-column px-lg-3 px-2 mt-3">
+                            {% if not setting_value('visiosoft.module.advs::hide_ad_cat') %}
+                                <a href="{{ adv.detail_url }}" class="product-name-hb text-dark">{{ adv.cat1_name }} /{{ adv.cat2_name }}</a>
+                            {% endif %}
+                            {% if showPrice %}
+                                <div class="product-price-hb font-weight-bold mt-2 text-dark">
+                                    {{ adv.price != '0' ? currency_format(adv.price,adv.currency) : trans('visiosoft.module.advs::field.free') }}
+                                    <span>TL</span>
```
Comment (muammertop) [inline addons/default/visiosoft/advs-module/resources/views/list/gallery.twig:29]: change to dynamic currency

**Review by muammertop — CHANGES_REQUESTED** [review]

**Review by fatihalp — CHANGES_REQUESTED** [review]
revert to addblock
change to dynamic currency

**Comment by fatihalp** [comment]
how you approved @spektra2147

File: composer.json [hunk 3]
```diff
@@ -93,8 +93,7 @@
         "filp/whoops": "~2.0",
         "fzaninotto/faker": "~1.4",
         "symfony/css-selector": "3.1.*",
-        "symfony/dom-crawler": "3.1.*",
-        "anomaly/installer-module": "~2.4.0"
```
Comment (muammertop) [inline composer.json:97]: revert to installer-module pls

File: addons/default/visiosoft/base-theme/resources/css/theme.css [hunk 4]
```diff
@@ -25,3 +25,66 @@
...
+    -webkit-line-clamp: 2;
+    height: 36px;
+    font-weight: normal;
+}
+
+.product-price-hb {
+    font-size: 18px;
+}
+
+.tl-span {
```
Comment (muammertop) [inline addons/default/visiosoft/base-theme/resources/css/theme.css:85]: where are you use this?

**Review by muammertop — APPROVED** [review]

**Comment by diashalabi** [comment]
I don't think it's a good idea to pull from interns @fatihalp Let's close this pull request as there are many things missing like the doping classes. This will affect all the customers who use openclassify, there is no margin for error in this repo :wink:

File: addons/default/visiosoft/advs-module/resources/views/list/gallery.twig [hunk 5]
```diff
@@ -5,48 +5,36 @@
     {% set showLocation = setting_value('visiosoft.module.location::home_page_location') %}
     {% set showDate = setting_value('visiosoft.theme.base::date_fields') %}
 
-    <div id="gallerying" class="d-flex flex-wrap">
+    <div class="row p-0 m-0 ">
 
         {{ addBlock('ad-list/partials/gallery',{'featured_advs': featured_advs, 'vars': _context})|raw }}
 
         {% for adv in advs %}
-            <div class="w-50 gallery-item-wrapper mb-1{{ adv.doping != null ? ' doping_type4' }}">
```
Comment (diashalabi) [inline addons/default/visiosoft/advs-module/resources/views/list/gallery.twig:13]: Why did you remove the doping check?

File: addons/default/visiosoft/advs-module/resources/views/list/gallery.twig [hunk 6]
```diff
@@ -5,48 +5,36 @@
...
+                <a href="{{ adv.detail_url }}" class="w-100 p-0 d-flex flex-column">
+                    <div class="d-flex flex-column product-hb border shadow-sm rounded">
+                        {{ addBlock('list/extra-actions', {'ad': adv, 'vars': _context}, ['favs', 'comparisons'])|raw }}
+
+                        <div class="product-image-hb">
+                            <img class="product-image-in" src="{{ adv.cover_photo }}" alt="{{ adv.name }}">
+                        </div>
+                        <div class="product-info-hb d-flex flex-column px-lg-3 px-2 mt-3">
+                            {% if not setting_value('visiosoft.module.advs::hide_ad_cat') %}
+                                <a href="{{ adv.detail_url }}" class="product-name-hb text-dark">{{ adv.cat1_name }} /{{ adv.cat2_name }}</a>
```
Comment (diashalabi) [inline addons/default/visiosoft/advs-module/resources/views/list/gallery.twig:23]: `<a>` tags should not be used inside `<a>` tags, please test before sending pull requests

File: addons/default/visiosoft/advs-module/resources/views/list/gallery.twig [hunk 7]
```diff
@@ -5,48 +5,36 @@
...
+                                <div class="product-price-hb font-weight-bold mt-2 text-dark">
+                                    {{ adv.price != '0' ? currency_format(adv.price,adv.currency) : trans('visiosoft.module.advs::field.free') }}
+                                </div>
+                            {% endif %}
+
                         </div>
                     </div>
-                    <div class="ml-4 mt-1">
-                        <h6 class="mb-1">
-                            <a href="{{ adv.detail_url }}">{{ adv.name }}</a>
```
Comment (diashalabi) [inline addons/default/visiosoft/advs-module/resources/views/list/gallery.twig:29]: Why removed the ad name?

**Review by diashalabi — CHANGES_REQUESTED** [review]

**Review by fatihalp — APPROVED** [review]

**Pull request merged**